extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Subscriptions Block.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Subscriptions;
+
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'subscriptions';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	jetpack_register_block( BLOCK_NAME );
+
+	Jetpack_Gutenberg::load_styles_as_required( FEATURE_NAME );
```

Comment:
I think it may be better to move this inside a `render_callback`, so we only enqueue the styles  when they're needed.

Reply:
Is there another way? There's no backend rendering of output for this block, so adding this call and placing the loading of styles here seems to override the frontend `save` call.

Reply:
Nevermind, I found you can just return the `$content` param. :+1: